doc-surrealdb_versioned_docs/version-latest/surrealql/functions/database/type.mdx

```diff
@@ -448,28 +448,26 @@ RETURN type::thing($tb, $id);
 
 <Since v="v2.0.0" />
 
-The `type::range` function converts a value into a record range.
+The `type::range` function converts a value into a record range. It accepts a single argument either a range or an array with two values will be converted into a range similar to [casting](/docs/surrealdb/surrealql/datamodel/casting).
```

Comment:
```suggestion
The `type::range` function converts a value into a record range. It accepts a single argument, either a range or an array with two values. If the argument is an array, it will be converted into a range,  similar to [casting](/docs/surrealdb/surrealql/datamodel/casting).
```